roles/backpack_kube/tasks/main.yml

```diff
@@ -1,48 +1,79 @@
 ---
+- name: Set facts
+  set_fact:
+    operator_namespace: backpack
+    es_server: "none"
+    es_port: 12345
+    metadata_privileged: "{{ metadata_privileged | default('false') }}"
+
 - name: Create Backpack Namespace
   k8s: 
-    name: backpack
+    name: "{{ operator_namespace }}"
     api_version: v1
     kind: Namespace
     state: present
     kubeconfig: "{{ kube_config }}"
 
-- name: Launch backpack daemonset
+- name: Convert kubeconfig to base64
+  shell: "base64 {{ kube_config }} | tr -d '\n'"
+  register: metadata_kubeconfig
+
+- name: Create UUID
```

Comment:
Just a suggestion to not use the `uuidgen` command, as it might not be present on all environments, but because we're using ansible we can possibly use something like https://github.com/cloud-bulldozer/ripsaw/blob/master/roles/uuid/tasks/main.yml.

Reply:
Good call. Updated.